### PR TITLE
TST: move misplaced modulo test

### PR DESCRIPTION
**Pull request by jbrockmendel**
Found this while looking at tests for reduction ops.  I'm wondering if we should collect+parametrize those over DataFrame/Series/Index/EA the same way we do for arithmetic.  Thoughts?

### CODE REVIEW

**Comment by gfyoung** [comment]
> I'm wondering if we should collect+parametrize those over DataFrame/Series/Index/EA the same way we do for arithmetic. Thoughts?

That seems like a reasonable proposal.

**Review by gfyoung — APPROVED** [review]
Not sure if we want to do anything else here, but the move LGTM though.

**Comment by jreback** [comment]
lgtm. ex @jorisvandenbossche comment (though certainly fine to do this later).

File: pandas/tests/arithmetic/test_numeric.py [hunk 1]
```diff
@@ -597,6 +597,44 @@ def test_operators_frame(self):
         tm.assert_series_equal(ts / ts, ts / df['A'],
                                check_names=False)
 
+    # TODO: this came from tests.series.test_analytics, needs cleannup and
+    #  de-duplication with test_modulo above
```
[the remark this answers is not in the document]
Comment (jbrockmendel) [reply pandas/tests/arithmetic/test_numeric.py]: Un-commited changes in my local branch make this a hassle I'd rather avoid.

[the remark this answers is not in the document]
Comment (jreback) [reply pandas/tests/arithmetic/test_numeric.py]: yeah let's not increase scope of things, plenty to do already.

**Pull request merged**